lang/python/python-docker/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-docker
+PKG_VERSION:=4.2.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=docker
+PKG_HASH:=ddae66620ab5f4bce769f64bcd7934f880c8abe6aa50986298db56735d0f722e
+
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+PYTHON3_PKG_SETUP_ARGS:=
+
+define Package/python3-docker
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=A Python library for the Docker Engine API
+  URL:=https://github.com/docker/docker-py
+  DEPENDS:=+python3-light +python3-paramiko +python3-six
```

Comment:
It breaks without paramiko:

`root@himawari ~/dockers/nextcloud $ docker-compose pull
Traceback (most recent call last):
  File "/usr/bin/docker-compose", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/__init__.py", line 3251, in <module>
  File "/__init__.py", line 3234, in _call_aside
  File "/__init__.py", line 3263, in _initialize_master_working_set
  File "/__init__.py", line 583, in _build_master
  File "/__init__.py", line 900, in require
  File "/__init__.py", line 786, in resolve                                                                                            
pkg_resources.DistributionNotFound: The 'paramiko>=2.4.2' distribution was not found and is required by docker
`

Reply:
Since it is docker-compose that [wants to use docker-py with ssh](https://github.com/docker/compose/blob/1.25.5/setup.py#L39), the python3-paramiko dependency should be added to the docker-compose package, not here.

Or alternatively, you can add an empty python3-docker-ssh package here that depends on python3-docker and python3-paramiko, then docker-compose can depend on python3-docker-ssh.